build.yaml

```diff
@@ -905,18 +1023,40 @@ job_type:
                    *exit_with_return_code_from_test ]
           }
       clean_repo: true
+      timeout: 30
     run_client_installation_on_OSX:
-      on_nodes_with_labels: 'osx'
+      on_nodes_with_labels: 'mesos-1GB'
       with_steps:
         - { type: 'shell',
-            cli: [ *add_shell_functions,
-                   *cleanup, *setup_venv, *setup_flocker_modules,
+            cli: [ *hashbang,
+                   *add_shell_functions,
+                   *new_vagrantfile_for_yosemite,
+
+                   *begin_build_sh_EOF,
+                   *hashbang,
```

Comment:
this is a cat <<EOF wrapper
the contents below will be placed inside a build.sh script which is copied to the vagrant box